azure-core/azure/core/pipeline_client_async.py

```diff
@@ -210,5 +211,8 @@ def merge(self, url, params=None, headers=None, content=None, form_content=None)
         await self._pipeline.__aenter__()
         return self
 
-    async def __aexit__(self, *exc_details):
-        await self._pipeline.__aexit__(*exc_details)
\ No newline at end of file
+    async def __aexit__(self, *args):
+        await self._pipeline.__aexit__(*args)
```

Comment:
I would expect `__aexit__()` to call `close()` rather than the other way around...